### ISSUE TITLE
photo has userInteraction = NO

### ISSUE THREAD

**Issue by Verteletsky** [evgenyneu/Auk#83]
* Library setup method: CocoaPods
* Version of the library. 
'moa', '> 10.0' 
'Auk', '> 9.0'
* Xcode version. Example: 10.1
* OS version. Example: iOS 12.1.1

I use the CollectionView inside of which there is a Cell, and in the xib file I have a ScrollView, how can I handle clicking on a photo, when the photo has userInteraction = NO, I can't get to it

**Comment by evgenyneu** [comment]
Hi @Verteletsky, what do you want to accomplish by clicking on a photo? [Here is how](https://github.com/evgenyneu/Auk#respond-to-image-tap) to respond to a tap, if that's what you need.

**Comment by Verteletsky** [comment]
@evgenyneu 
да, я хочу обработать нажатие, но у фотографии userInteraction = no, потому что у меня scrollView находится в ячейке в коллекции

Yes, I want to process the click, but the photo has userInteraction = no, because my scrollView is in a cell in the collection

**Comment by evgenyneu** [comment]
@Verteletsky there is a section in the readme called [Respond to image tap](https://github.com/evgenyneu/Auk#respond-to-image-tap) that might help. Let me know if that works for you. 

PS. Good to see Russian language. :)
